apiserver/facades/controller/crossmodelrelations/crossmodelrelations_test.go

```diff
@@ -136,7 +137,7 @@ func (s *crossmodelRelationsSuite) setupAPI(c *gc.C) {
 		s.st, fw, s.resources, s.authorizer,
 		authContext, s.secretService, s.modelConfigService, egressAddressWatcher, relationStatusWatcher,
 		offerStatusWatcher, consumedSecretsWatcher,
-		loggertesting.WrapCheckLog(c),
+		loggertesting.WrapCheckLog(c), model.UUID(s.st.ModelUUID()),
```

Comment:
Same question we want to get off state.

Reply:
this is just the mock state inside a test, but OK